Files/UserControls/Widgets/DrivesWidget.xaml

```diff
@@ -167,18 +167,19 @@
                                 VerticalAlignment="Stretch"
                                 ColumnSpacing="12">
                                 <Grid.ColumnDefinitions>
-                                    <ColumnDefinition Width="44" />
+                                    <ColumnDefinition Width="Auto" />
                                     <ColumnDefinition Width="*" />
                                 </Grid.ColumnDefinitions>
                                 <Grid.RowDefinitions>
+                                    <RowDefinition Height="*" />
                                     <RowDefinition Height="Auto" />
```

Comment:
Can you hard code the height for the progress bar row? This ensures there is a enough padding so it doesn't look too crowded.

Reply:
@yaichenbaum There's no need for this. I tested it.

Reply:
If you look at the comparison, the progress bar has the wrong padding.

Reply:
@yaichenbaum The padding difference is normal because the text above and below is larger. Further, if we wanted to increase the padding, we wouldn't want to hardcode it into the row's height.

Reply:
@duke7553 In that case, can you adjust the padding so that it matches the previous design? Additionally, I just noticed you set the others rows to stretch, is there any reason you choose that over `auto`?

Reply:
@yaichenbaum Users will not notice the difference pertaining to the padding because it looks essentially the same when text is at 100 percent scaling. Remember, this PR fixes a critical issue with text being clipped when scaled for accessibility. 

**It is never a good idea to hardcode values.** In fact, Microsoft's guidelines for accessibility advise against this. Further, there is no visual or functional difference in choosing stretch for the row height instead of what existed previous except that the margin is now applied neatly to everything because it is stretched.